python/python-core/basic-file-manipulation/how-to-open-a-file-object.md

```diff
@@ -80,15 +80,15 @@ print("The file mode is: ", obj.mode)
 Suppose we want to append something to a file and print its content. Fill the gaps accordingly:
 
 ```python
-file = open('practice.py', '???')
+file = ???('practice.py', '???')
 file.write('Append this')
 file.???(0)
-print(file.???())
+print(file.read())
```

Comment:
since we teach `read` in the next insight shouldn't we remove this line entirely?

Reply:
removed it and modified the sentence to not contain "and print it out"